hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/jdbc/DMLProcessorSupplier.java

```diff
@@ -40,34 +40,38 @@
 import static java.util.Collections.singletonList;
 import static java.util.Objects.requireNonNull;
 
-public class UpdateProcessorSupplier
+public class DMLProcessorSupplier
```

Comment:
nit: DmlProcessorSupplier, same as JdbcSqlConnector not JDBCSQLConnector

Reply:
85371ae0ddc